django/db/models/base.py

```diff
@@ -387,7 +387,13 @@ class ModelState:
     # on the actual save.
     adding = True
     fields_cache = ModelStateFieldsCacheDescriptor()
+    prefetch_related_peers = ()
 
+    def __getstate__(self):
+        """Hook to allow choosing the attributes to pickle."""
+        res = dict(self.__dict__)
```

Comment:
`self.__dict__.copy()` is more explicit

also it looks like `obj_dict` is the 'standard' name for `self.__dict__.copy()` in django `__getstate__` methods.